### ISSUE TITLE
qmake generator has problems with spaces

### ISSUE THREAD

**Issue by mnowy** [conan-io/conan#578]
If you have space in your OS login, ie: "Name Surname", qmake generator generates malformed conanbuildinfo.pri file.

It generates entries like this: 
`CONAN_INCLUDEPATH += C:/Users/Name Surname/.conan/data/library/1.0.0/conanUser/stable/package/b3fac16af3318594047b089aa411f781c8d81783/include`
and because of space in path, it should be in quotas like this:
`CONAN_INCLUDEPATH += "C:/Users/Name Surname/.conan/data/library/1.0.0/conanUser/stable/package/b3fac16af3318594047b089aa411f781c8d81783/include"`

**Comment by memsharded** [comment]
What would be the correct syntax for multiple entries of include_paths?

**Comment by memsharded** [comment]
Also, I guess the problem would be the same for lib paths: 
```
CONAN_LIBS += -L"C:/path with spaces/folder" \
                           -L"C:/path2 with spaces/folder" 
```

**Comment by mnowy** [comment]
Woow you're quick! :)

Correct syntax for multiple entries of include_paths would be similar to your `CONAN_LIBS` example but I guess you know that already.

Will that fix be released in next conan version?

**Comment by lasote** [comment]
@mnowy yes, we hope to release today or tomorrow (we are having very long delays with travis OSx builds)

**Comment by memsharded** [comment]
Will be released in 0.14
